src/index.html

```diff
@@ -9,6 +9,68 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="logo">
+        <a href="#home">
+          <img
+            src="./images/logo.png"
+            alt="home"
```

Comment:
alt attribute should be descriptive, check out checklist point 7
```suggestion
            alt="Moyo logo"
```